docs/src/userguide/subsetting_a_cube.rst

```diff
@@ -332,40 +332,135 @@ on bounds can be done in the following way::
 The above example constrains to cells where either the upper or lower bound occur
 after 1st January 2008.
 
+.. _cube_masking:
+
 Cube Masking
 --------------
 
+Masking a cube allows you to hide unwanted data points without changing the 
+shape or size of the cube.  This can be achieved by two methods:
+
+1. Masking a cube using a boolean mask array via :func:`iris.util.mask_cube`.
+2. Masking a cube using a shapefile via :func:`iris.util.mask_cube_from_shape`.
+
+.. _masking-from-boolean:
+
+Masking a cube using a boolean mask array
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :func:`iris.util.mask_cube` function allows you to mask unwanted data points
+in a cube using a boolean mask array. The mask array must have the same shape as
+the data array in the cube, with ``True`` values indicating points to be masked.
+
+For example, the mask could be based on a threshold value. In the following
+example we mask all points in a cube where the air potential temperature is
+greater than 290 K.
+
+    >>> filename = iris.sample_data_path('uk_hires.pp')
+    >>> cube_temp = iris.load_cube(filename, 'air_potential_temperature')
+    >>> print(cube_temp.summary(shorten=True))
+    air_potential_temperature / (K)     (time: 3; model_level_number: 7; grid_latitude: 204; grid_longitude: 187)
+    >>> type(cube_temp.data)
+    <class 'numpy.ndarray'>
+
+Note that this example cube has 4 dimensions: time and model level number in
+addition to grid latitude, and grid longitude. The data array associated with
+the cube is a regular :py:class:`numpy.ndarray`.
+
+We can build a boolean mask array by applying a condition to the cube's data
+array:
+
+    >>> mask = cube_temp.data > 290
+    >>> cube_masked = iris.util.mask_cube(cube_temp, mask)
+    >>> print(cube_masked.summary(shorten=True))
+    air_potential_temperature / (K)     (time: 3; model_level_number: 7; grid_latitude: 204; grid_longitude: 187)
+    >>> type(cube_masked.data)
+    <class 'numpy.ma.MaskedArray'>
+
+The masked cube will have the same shape and coordinates as the original cube,
+but the data array now includes an associated boolean mask, and the cube's
+`data` property is now a :py:class:`numpy.ma.MaskedArray`.
+
 .. _masking-from-shapefile:
 
 Masking from a shapefile
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Often we want to perform some kind of analysis over a complex geographical feature e.g.,
+Often we want to perform analysis over a complex geographical feature e.g.,
 
-- over only land/sea points
+- over only land or sea points
```

Comment:
I had no problem with the original voice here, but I guess the changes are equally good